.travis.yml

```diff
@@ -6,8 +6,7 @@ services:
 python:
 - '3.6'
 install:
-# - pip install shybrid
-- pip install mountainlab_pytools pynwb==1.3.3 pyopenephys kbucket h5py MEArec==1.4.1 exdir ruamel.yaml nixio neo>=0.8 hdf5storage
```

Comment:
You need to comment our the shybrid install and point MEArec to 1.4.1, as it was in master

Reply:
Done! Current version of the travis.yaml should be identical to the master on SpikeInterface, but with bs4 and lxml added.